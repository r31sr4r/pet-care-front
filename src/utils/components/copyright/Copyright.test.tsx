import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { Copyright } from './Copyright';

describe('Copyright', () => {
    it('should render successfully', () => {
        const { asFragment } = render(<Copyright />, {
        	wrapper: BrowserRouter, 
        });
        expect(asFragment()).toMatchSnapshot();                
    });
});