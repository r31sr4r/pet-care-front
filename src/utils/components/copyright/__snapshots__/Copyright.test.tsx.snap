// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright should render successfully 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
  >
    Copyright Â© 
    <a
      class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1cghntr-MuiTypography-root-MuiLink-root"
      href="https://www.onewayinnovation.com.br/"
      target="_blank"
    >
      One Way Innovation
    </a>
     2022.
  </p>
</DocumentFragment>
`;
